pkgs/development/python-modules/thorlabspm100/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytestCheckHook
+, pytest-runner
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  version = "1.2.2";
+  pname = "ThorlabsPM100";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "881cec8149b17c77d4af0097ee3fdb40eaeb618c876d49f1bc42c87611737967";
+  };
+
+  propagatedBuildInputs = [
+    setuptools-scm
+    pytest-runner
```

Comment:
```suggestion
```
`pytest-runner` does not seem to be used in the upstream.
Would you please add `unittestCheckHook` to `nativeCheckInputs` as an alternative?

Reply:
also it does not work in nixpkgs and is rather useless